### PR TITLE
[EN] 09-3.md

### PR DESCRIPTION
**Pull request by wh629**
Hi Alf,

Here are the notes for Week 9's practicum. I'm fairly new to github, so I hope this works out ok.

Just some notes, I forked the repository a little over a week ago so some of the other files are old. Please let me know if there's anything that needs to be changed.

Thanks!

### CODE REVIEW

File: docs/en/week09/09-3.md [hunk 1]
```diff
@@ -0,0 +1,230 @@
...
+            nn.ConvTranspose2d(    ngf,      nc, 4, 2, 1, bias=False),
+            nn.Tanh()
+            # state size. (nc) x 64 x 64
+        )
+
+    def forward(self, input):
+        if input.is_cuda and self.ngpu > 1:
+            output = nn.parallel.data_parallel(self.main, input, range(self.ngpu))
+        else:
+            output = self.main(input)
```
Comment (Atcold) [inline docs/en/week09/09-3.md:115]:
```suggestion
        output = self.main(input)
```

File: docs/en/week09/09-3.md [hunk 2]
```diff
@@ -0,0 +1,230 @@
...
+#### Discriminator
+
+1. It is important to use `nn.LeakyReLU` as the activation function to avoid killing the gradients in negative regions. Without these gradients, the generator will not receive updates.
+2. At the end of the sequential, the discriminator uses `nn.Sigmoid()` to classify the input.
+
+```python
+class Discriminator(nn.Module):
+    def __init__(self, ngpu):
+        super(Discriminator, self).__init__()
+        self.ngpu = ngpu
```
Comment (Atcold) [inline docs/en/week09/09-3.md:128]:
```suggestion
    def __init__(self):
        super().__init__()
```

File: docs/en/week09/09-3.md [hunk 3]
```diff
@@ -0,0 +1,230 @@
...
+            nn.Conv2d(ndf * 8, 1, 4, 1, 0, bias=False),
+            nn.Sigmoid()
+        )
+
+    def forward(self, input):
+        if input.is_cuda and self.ngpu > 1:
+            output = nn.parallel.data_parallel(self.main, input, range(self.ngpu))
+        else:
+            output = self.main(input)
+
```
Comment (Atcold) [inline docs/en/week09/09-3.md:155]:
```suggestion
        output = self.main(input)
```
